### ISSUE TITLE
Extend tappable title area

### ISSUE THREAD

**Issue by nascentt** [QuantumBadger/RedReader#1085]
Hard to describe but currently when tapping the post title or the subresdit title will go back to the previous screen.

This works great when using the phone right handed with one hand.

However if a title is very shot, or the subresdit name is short. It's difficult to read far enough left to tap it with the right hand.

It'd be great if that whole blank area where a longer title should be was still clickable. Even if optionally.

**Comment by nascentt** [comment]
Others in r/redreader have asked for this https://reddit.com/comments/150d9m7

**Comment by pcamilo89** [comment]
![image](https://github.com/QuantumBadger/RedReader/assets/655782/7a7e7af3-4425-4e7b-8634-a592a086c2e7)
this is also a problem when you open a post properties dialog, to close it you have to hit the blue area just on top of the letters, it should be a larger area like the one marked in red

**Comment by QuantumBadger** [comment]
@nascentt / @pcamilo89 I think there are three separate requests here, the linked reddit thread is about posts in the post list.

**Comment by nascentt** [comment]
All three things are about being able to tap the areas without text.

**Comment by QuantumBadger** [comment]
Yeah, they're three different features in different parts of the codebase, and they all need a different response:

1. This ticket (bigger tappable area in actionbar) -- the current behaviour is deliberate, but I'm happy to accept a PR for this if it's off by default
3. @pcamilo89's comment: this is just how dialogs behave on Android, the button doesn't take up the full width of the dialog. I wouldn't want us to hack the dialog theme to expand this, but moving the button to the right hand side (when the app isn't in left hand mode) would be reasonable
4. The linked reddit thread: this seems to be asking that post title text should have its own unique action, whereas actually the post view is two big buttons next to each other -- a big button on the left which has all the post details inside, and a button on the right for comments. I'd maybe consider having the thumbnail being a separate, customizable button (and adding a preference for always showing the thumbnail square), but in general this one seems to be a case of "please make the app exactly like this other app I used before".

**Comment by dsibilio** [comment]
> 3\. The linked reddit thread: this seems to be asking that post title text should have its own unique action, whereas actually the post view is two big buttons next to each other -- a big button on the left which has all the post details inside, and a button on the right for comments. I'd maybe consider having the thumbnail being a separate, customizable button (and adding a preference for always showing the thumbnail square), but in general this one seems to be a case of "please make the app exactly like this other app I used before".

@QuantumBadger, it kind of makes sense to me that if someone turned on the `Post title opens comments` option, they would half-expect also the subtitle text to respect the same behaviour. I know for a fact that's definitely what I expected when I turned it on. Misclicking the subtitle by a few pixels and opening the content instead of the comments is painful. If it's not acceptable to change the current behaviour of that option, I wonder if it would be possible to at least add one more setting to also open comments via clicking the Subtitle _(and if possible the empty area next to the title/subtitle as well)_? E.g. `Post subtitle opens comments`.

_P.S.: for whoever is willing to make their own custom build in the meanwhile to extend the title behaviour to the subtitle as well, you can simply edit
https://github.com/QuantumBadger/RedReader/blob/master/src/main/java/org/quantumbadger/redreader/views/RedditPostView.java#L292-L294 to look like the following instead:_
```java
		if(postTitleOpensPost) {
			title.setOnClickListener(v -> fragmentParent.onPostCommentsSelected(mPost));
			subtitle.setOnClickListener(v -> fragmentParent.onPostCommentsSelected(mPost));
		}
```
_It should work straight away, but note that it doesn't cover the empty space next to the title/subtitle though as I didn't quite investigate that yet._

**Comment by QuantumBadger** [comment]
@dsibilio Ah, thank you for the clarification, I'd misunderstood the report. That's a bug, it looks like the view which we actually want to set the click listener on has ID `reddit_post_textLayout`.
